### DISCUSSION TITLE
Error about Submitted Files("path"...)

### DISCUSSION THREAD

**Discussion by LiuZhihhxx** [PaddlePaddle/PaddleSpatial#168] [BaiduKDDCup2022] [answered]
A submission error occured as shown in the following figure:

![image](https://user-images.githubusercontent.com/73637604/173213581-5b427deb-d792-44e2-98ac-28efbfbd28df.png)

Actually, I just followed the instructions posted on _https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation_

I wonder if the error message is not displayed completely due to the limitted interface.
 
In my submission zip file, the extracted items be like :
 ./MyFolderName 
 | --- __init__.py         
 | --- predict.py  
 | --- prepare.py
 | --- ... 
 | --- ./checkpoints (including a filefolder for trained models, named 'wtbdata_245days.csv_tMS_i144_o288_ls2_train200_val45' as in the figure above)

Any help would be appreciated!

**Comment by seeingisbelieveing** [comment, marked as answer]
Dear participant, could you please provide your team name and submission date? We will check it, Thx.

**Reply by LiuZhihhxx** [reply]
Team CSU123
The last submission with this error was at 16:00, June 13th.
Thx a lot.

**Reply by seeingisbelieveing** [reply]
The full error log should be "Exception: Err: The ``path`` (/tmp/tmpjdbggla3/checkpoints/wtbdata_245days.csv_tMS_i144_o288_ls2_train200_val45/model_0) to load model not exists.! (paddle6121935.zip)"

**Comment by LiuZhihhxx** [comment]
OK, I'll check my scripts. Thanks for your kindness.
